### ISSUE TITLE
Fail to specify reporter output that contains spaces in path

### ISSUE THREAD

**Issue by Tapchicoma** [shyiko/ktlint#177]
It was requested in [ktlint-gradle](https://github.com/JLLeitschuh/ktlint-gradle) plugin to support reporters output path that contains spaces.

I've tried following approaches to check if it supported on Ktlint side:
```bash
$ ~/tmp/ktlint --debug --reporter=plain,output="/home/user/some dir with spaces/samples/android-app/build/reports/ktlint/ktlint-debugAndroidTest.txt" "src/**/*.kt"
Error: "--reporter=plain,output=/home/user/some dir with spaces/samples/android-app/build/reports/ktlint/ktlint-debugAndroidTest.txt" is not a valid option
```

```bash
$ ~/tmp/ktlint --debug --reporter=plain,output=/home/user/some dir with spaces/samples/android-app/build/reports/ktlint/ktlint-debugAndroidTest.txt "src/**/*.kt"
[DEBUG] Discovered ruleset "standard"
[DEBUG] Discovered reporter "plain"
[DEBUG] Discovered reporter "json"
[DEBUG] Discovered reporter "checkstyle"
[DEBUG] Initializing "plain" reporter with {verbose=false, color=false}, output=/home/user/some
[DEBUG] Checking /home/user/some dir with spaces/samples/android-app/src/androidTest/java/org/jlleitschuh/gradle/ktlint/android/ExampleInstrumentedTest.kt
[DEBUG] Checking /home/user/some dir with spaces/samples/android-app/src/main/java/org/jlleitschuh/gradle/ktlint/android/ItemDetailActivity.kt
[DEBUG] Checking /home/user/some dir with spaces/samples/android-app/src/main/java/org/jlleitschuh/gradle/ktlint/android/ItemDetailFragment.kt
[DEBUG] Checking /home/user/some dir with spaces/samples/android-app/src/main/java/org/jlleitschuh/gradle/ktlint/android/ItemListActivity.kt
[DEBUG] Checking /home/user/some dir with spaces/samples/android-app/src/main/java/org/jlleitschuh/gradle/ktlint/android/dummy/DummyContent.kt
[DEBUG] Checking /home/user/some dir with spaces/samples/android-app/src/test/java/org/jlleitschuh/gradle/ktlint/android/ExampleUnitTest.kt
[DEBUG] 983ms / 6 file(s) / 0 error(s)
```

```bash
$  ~/tmp/ktlint --reporter=plain,output=/home/user/some\ dir\ with\ spaces/samples/android-app/build/reports/ktlint/ktlint-debugAndroidTest.txt "src/**/*.kt"         
Error: "--reporter=plain,output=/home/user/some dir with spaces/samples/android-app/build/reports/ktlint/ktlint-debugAndroidTest.txt" is not a valid option
```

**Comment by eskatos** [comment]
This should work:
```
$ ~/tmp/ktlint "--reporter=plain,output=/home/user/some dir with spaces/samples/android-app/build/reports/ktlint/ktlint-debugAndroidTest.txt" "src/**/*.kt" 
```
that is, quoting the whole "--reporter=..." argument.

**Comment by Tapchicoma** [comment]
Hm, I also getting error in this case as well:
```bash
$ ~/tmp/ktlint --debug "--reporter=plain,output=/home/user/some dir with spaces/samples/android-app/build/reports/ktlint/ktlint-debugAndroidTest.txt" "src/**/*.kt"
Error: "--reporter=plain,output=/home/user/some dir with spaces/samples/android-app/build/reports/ktlint/ktlint-debugAndroidTest.txt" is not a valid option

```

**Comment by shyiko** [comment]
Fixed in [0.20.0](https://github.com/shyiko/ktlint/releases/tag/0.20.0).
